### PR TITLE
feat(pie-docs): create hero section and align styling

### PR DESCRIPTION
**Pull request by raoufswe**
[Added] hero section and align styling
[Updated] pie logo to the new version

### CODE REVIEW

File: apps/pie-docs/src/_includes/content-page.njk [hunk 1]
```diff
@@ -11,15 +11,19 @@
             <p class="c-contentPage-header-description">
                 {{ description }}
             </p>
+            {% if heroCtaTitle and heroCtaHref %}
+                <a class='c-contentPage-header-cta' href={{ heroCtaHref }}>{{ heroCtaTitle }}
```
Comment (jamieomaguire) [inline apps/pie-docs/src/_includes/content-page.njk:15]: I've noticed this is an absolute link to prod on the preview, you might need to use the nunjucks `| url` filter here. Please look at other examples of it's usage in the project :)

**Comment by jamieomaguire** [comment]
Screenshot 2023-02-24 at 09 12 32

To save back and forth with Jordi, please could you show him examples of this at awkward screen sizes (large and small) and ensure he's happy before merging?

File: apps/pie-docs/src/assets/styles/components/_contentPage.scss [hunk 2]
```diff
@@ -39,29 +39,32 @@
...
-    @include f.media('>wide') {
-        max-width: 55%;
+    .c-contentPage-header-cta {
+        display: inline-block;
+        background-color: f.$color-interactive-brand;
+        color: f.$color-container-default;
+        font-size: f.$font-interactive-m-size;
+        font-weight: f.$font-interactive-m-weight;
+        border-radius: f.$radius-rounded-e;
+        padding: 10px f.$spacing-e;
```
Comment (jamieomaguire) [inline apps/pie-docs/src/assets/styles/components/_contentPage.scss:65]: it's uncommon for us to deviate away from the regular padding values we have (multiples of 8) - is this definitely what we want? :)

Comment (jamieomaguire) [reply apps/pie-docs/src/assets/styles/components/_contentPage.scss:65]: We already have a button/cta style from Fozzie - if this is that then I'd recommend using `o-btn` here

**Comment by jamieomaguire** [comment]
Screenshot 2023-02-24 at 09 26 47
Screenshot 2023-02-24 at 09 27 04

Just checking the newer whitespace on the left and right is as designed etc

File: apps/pie-docs/src/assets/styles/components/_contentPage.scss [hunk 3]
```diff
@@ -39,29 +39,32 @@
...
-    }
-}
+    @include f.media('>=mid') {
+        display: flex;
+        gap: f.$spacing-f;
 
-.c-contentPage-header--hero .c-contentPage-header-heading,
-.c-contentPage-header--hero .c-contentPage-header-description {
-    @include f.media('>=narrowMid') {
-        max-width: 60%;
```
Comment (jamieomaguire) [inline apps/pie-docs/src/assets/styles/components/_contentPage.scss:60]: I'm unsure if we want to remove this/need to replace it with something else

File: apps/pie-docs/src/assets/styles/components/_contentPage.scss (hunk 2 above)
Comment (raoufswe) [reply apps/pie-docs/src/assets/styles/components/_contentPage.scss:65]: Oh, i didn't know about this, changed!

File: apps/pie-docs/src/assets/styles/components/_contentPage.scss (hunk 3 above)
Comment (raoufswe) [reply apps/pie-docs/src/assets/styles/components/_contentPage.scss:60]: keeping these declarations won't align the spacing between the content and the image to the new design.
we only should have a gap of 32px and the content should stretch the whole container.

Comment (raoufswe) [reply apps/pie-docs/src/assets/styles/components/_contentPage.scss:60]: had a chat with @jordifornellsJET and we believe we should add a max-width of 500px for the content 💯

File: apps/pie-docs/src/index.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
 ---
 layout: layouts/page-layout.njk
-title: Principles for Interfaces and Experiences
-description: PIE is the global design system for Just Eat Takeaway.com, powering all our products.
+title: PIE Design System
+description: Just Eat Takeaway.com’s global principles for interfaces and experiences
 heroImageSrc: home-hero.svg
+heroImageWidth: 424px
```
Comment (jamieomaguire) [inline apps/pie-docs/src/index.md:6]: the designs actually show this image scaling down as the screensize shrinks - might not be ideal to hard code the absolute size of it. Maybe a max width and an aspect ratio?

Comment (jordifornellsJET) [reply apps/pie-docs/src/index.md:6]: @jamieomaguire @raoufswe It did look good with a fixed size for the image, but I would like to see it scaling down too if possible and make a decision based on the results?

Comment (raoufswe) [reply apps/pie-docs/src/index.md]: the image will scale dynamically now 💯

**Review by fernandofranca — APPROVED** [review]

File: apps/pie-docs/src/_includes/content-page.njk [hunk 5]
```diff
@@ -1,25 +1,34 @@
...
+                    {{ description }}
+                </p>
+                {% if heroCtaTitle and heroCtaHref %}
+                    <a class='o-btn' href={{ heroCtaHref | url }}>{{ heroCtaTitle }}
+                    </a>
+                {% endif %}
+            </div>
+            {% if heroImageSrc %}
+                <div>
+                    <img class="u-showAboveMid" src={{ heroSrc }} alt="{{ heroImageAlt }}"/>
```
Comment (jamieomaguire) [inline apps/pie-docs/src/_includes/content-page.njk:20]: Is an alt necessary here? It's a decorative image being displayed next to a description of the website, so I wonder what an alt is really adding to the screen reader UX here

Comment (jamieomaguire) [reply apps/pie-docs/src/_includes/content-page.njk:20]: wondering if it should be an alt vs defining the role is decoration etc

**Review by jamieomaguire — APPROVED** [review]

**Comment by FayeCarter** [comment]
image
I think there's a background colour in `c-contentPage-header` that needs to be removed. Not sure how well it'll show up in the screenshot

**Review by FayeCarter — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Comment by jamieomaguire** [comment]
Looks good - cracking job @raoufswe :)

File: apps/pie-docs/src/_includes/content-page.njk (hunk 5 above)
Comment (jamieomaguire) [reply apps/pie-docs/src/_includes/content-page.njk]: awesome! cheers

**Review by jamieomaguire — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**